suites/pacific/rgw/tier-1-extn_rgw_multisite-secondary-to-primary.yaml

```diff
@@ -352,8 +352,34 @@ tests:
       polarion-id: CEPH-83575360
       module: sanity_rgw_multisite.py
       clusters:
-        ceph-sec:
+        ceph-pri:
```

Comment:
Looks good, But IMO, this suite file contains all the test-cases that are executed in secondary site and validated from primary site.

Reply:
@ckulal that is correct , but does it make sense to create a new tier-1 suite yaml for just 2 test cases?

Reply:
IMO, we can create new suites or else we can change suite name accordingly,
if we make the suite name change please make sure to change the names in respective metadata file also.